core/lib/scmi-server/conf-optee-stm32mp1.mk

```diff
@@ -0,0 +1,18 @@
+_CFG_SCPFW_MOD_RESET_DOMAIN := y
```

Comment:
Why do you use a `_CFG_` prefix instead of `CFG_`?

Reply:
I initially did not want to add a lot of `CFG_` if not necessary. These switches are set from the platform and I don't expect a build env to override the platform selected config. I though `_CFG` would apply.

Reply:
In my view we use a `_CFG` prefix for internal helper variables that are derived by some logic from other flags, that is, they are normally not set directly in a configuration. I'd say when in doubt, use an ordinary `CFG` prefix instead.